File: src/main/java/Main/ItemClasses/Weapons/LongRangeWeapon.java
package Main.ItemClasses.Weapons;

public class LongRangeWeapon extends Weapon{
    public LongRangeWeapon(String name, String description, int weight, int valueInGold, int amount, String damage) {
        super(name, description, weight, valueInGold, amount, damage);
        this.standardIconName = "Longbow.png";
    }

    public LongRangeWeapon() {
        super();
        this.standardIconName = "Longbow.png";
    }
}

File: src/main/kotlin/main/ItemClasses/Weapons/LongRangeWeapon.kt
package main.ItemClasses.Weapons

import kotlinx.serialization.Serializable

@Serializable
class LongRangeWeapon() : Weapon() {
    init {
        standardIconName = "Longbow.png"
    }

    constructor(name: String, description: String, weight: Int, valueInGold: Int, amount: Int, damage: String) : this() {
        this.name = name
        this.description = description
        this.weight = weight
        this.valueInGold = valueInGold
        this.amount = amount
        this.damage = damage
    }
}
